code/lib/codemod/package.json

```diff
@@ -20,21 +20,18 @@
   },
   "license": "MIT",
   "sideEffects": false,
+  "type": "module",
   "exports": {
     ".": {
       "types": "./dist/index.d.ts",
-      "node": "./dist/index.js",
-      "require": "./dist/index.js"
+      "default": "./dist/index.js"
     },
-    "./dist/transforms/csf-2-to-3.js": "./dist/transforms/csf-2-to-3.js",
-    "./dist/transforms/csf-hoist-story-annotations.js": "./dist/transforms/csf-hoist-story-annotations.js",
-    "./dist/transforms/find-implicit-spies.js": "./dist/transforms/find-implicit-spies.js",
-    "./dist/transforms/upgrade-deprecated-types.js": "./dist/transforms/upgrade-deprecated-types.js",
-    "./dist/transforms/upgrade-hierarchy-separators.js": "./dist/transforms/upgrade-hierarchy-separators.js",
-    "./package.json": "./package.json"
+    "./package.json": "./package.json",
+    "./transforms/csf-2-to-3": "./dist/transforms/csf-2-to-3.js",
+    "./transforms/find-implicit-spies": "./dist/transforms/find-implicit-spies.js",
+    "./transforms/upgrade-deprecated-types": "./dist/transforms/upgrade-deprecated-types.js",
+    "./transforms/upgrade-hierarchy-separators": "./dist/transforms/upgrade-hierarchy-separators.js"
```

Comment:
✨

Reply:
not true, you can easily have entries that don't have an `exportEntries` property. eg. see 

https://github.com/storybookjs/storybook/blob/b6b6f15ba6d36ac6a0bacd274da1cb00e13528a5/scripts/build/entries/index.ts#L149-L152